### PR TITLE
RWA-1143 fixed failing tests

### PR DESCRIPTION
**Pull request by vgulapally**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-1143

### Change description ###
corrected failing tests as per IAC DMN changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by dharmendrak — COMMENTED** [review]
Looks good to me, however would request Ignore of R1 tests need to reset

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostUnclaimByIdControllerTest.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
 import static uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.CamundaVariableDefinition.REGION;
 import static uk.gov.hmcts.reform.wataskmanagementapi.services.SystemDateProvider.DATE_TIME_FORMAT;
 
-@Ignore("Release 1 test class")
+//@Ignore("Release 1 test class")
```
Comment (BenNzewi) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostUnclaimByIdControllerTest.java:25]: can this ignored comments be removed?

**Review by BenNzewi — COMMENTED** [review]
There are a number of commented ignored tests. Can this comments be removed?

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostUnclaimByIdControllerTest.java (hunk 1 above)
Comment (Lee365) [reply src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostUnclaimByIdControllerTest.java]: Removed

**Review by rasikasri — APPROVED** [review]

**Pull request merged**